tests/Adapters/Microsoft.Bot.Builder.Adapters.Slack.TestBot/DeploymentTemplates/template-with-new-rg.json

```diff
@@ -117,6 +118,9 @@
         "[resourceId('Microsoft.Resources/resourceGroups/', parameters('groupName'))]"
       ],
       "properties": {
+        "expressionEvaluationOptions": {
+          "scope": "outer"
+        },
```

Comment:
This is the default behavior. Do not need to add it.

Reply:
True. I left this in so I could remember where this scope setting goes. But I will take it out.